UI/forms/OBSBasicSettings.ui

```diff
@@ -1849,6 +1849,91 @@
+              <item>
+               <widget class="QGroupBox" name="simulcastGroupBox">
+                <property name="title">
+                 <string>Basic.Settings.Stream.SimulcastLabel</string>
+                </property>
+                <layout class="QVBoxLayout" name="verticalLayout_35">
+                 <property name="leftMargin">
+                  <number>9</number>
+                 </property>
+                 <property name="topMargin">
+                  <number>2</number>
+                 </property>
+                 <property name="rightMargin">
+                  <number>9</number>
+                 </property>
+                 <property name="bottomMargin">
+                  <number>9</number>
+                 </property>
+                 <item>
+                  <layout class="QHBoxLayout" name="horizontalLayout_33">
+                   <item>
+                    <spacer name="horizontalSpacer_33">
+                     <property name="orientation">
+                      <enum>Qt::Horizontal</enum>
+                     </property>
+                     <property name="sizeType">
+                      <enum>QSizePolicy::Fixed</enum>
+                     </property>
+                     <property name="sizeHint" stdset="0">
+                      <size>
+                       <width>170</width>
+                       <height>10</height>
+                      </size>
+                     </property>
+                    </spacer>
+                   </item>
+                   <item>
+                    <widget class="QLabel" name="simulcastInfo">
+                     <property name="text">
+                      <string>Basic.Settings.Stream.SimulcastInfo</string>
+                     </property>
+                     <property name="textFormat">
+                      <enum>Qt::RichText</enum>
+                     </property>
+                     <property name="wordWrap">
+                      <bool>true</bool>
+                     </property>
+                    </widget>
+                   </item>
+                  </layout>
+                 </item>
+                 <item>
+                  <layout class="QFormLayout" name="formLayout_39">
+                   <property name="fieldGrowthPolicy">
+                    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
+                   </property>
+                   <item row="1" column="0">
+                    <widget class="QLabel" name="simulcastTotalLayersLabel">
+                     <property name="text">
+                      <string>Basic.Settings.Stream.SimulcastTotalLayers</string>
+                     </property>
+                    </widget>
+                   </item>
+                   <item row="1" column="1">
+                    <layout class="QHBoxLayout" name="horizontalLayout_34" stretch="0,0">
+                     <item>
+                      <widget class="QSpinBox" name="simulcastTotalLayers">
```

Comment:
Looks like the encoding size / bitrate information box from the simulcast RFC is missing

Reply:
@DDRBoxman I forgot to remove this from RFC. I was told this is too busy/too much UI.

Reply:
Updated RFC

Reply:
I think that this ends up with us back in a similar situation to the opaque checkbox we didn't like in the first place.

Can we add a link below there that pops up a dialog that has static text to explain how the different layers will be calculated and what else to expect.